src/proc_fuse.c

```diff
@@ -999,6 +1000,16 @@ static int proc_stat_read(char *buf, size_t size, off_t offset,
 
 		curcpu++;
 
+		if (cgroup_ops->can_use_cpuview(cgroup_ops) && opts && opts->use_cfs)
+			use_view = true;
+		else
+			use_view = false;
+
+		if (use_view)
+			cpu_to_render = curcpu;
+		else
+			cpu_to_render = physcpu;
+
```

Comment:
Looks like `use_view` is only defined and used for the immediately following if statement.

Could we drop that extra variable and just directly set `cpu_to_render` instead?

Reply:
done, thanks